index.js

```diff
@@ -12,6 +12,7 @@ const sseMiddleware = require('./middlewares/sse');
 
 const { USERS_TABLE, selectFirstUser } = require('./services/db');
 const { middleware: apiTokenMiddleware } = require('./middlewares/apiToken');
+const stringify = require('json-stringify-safe');
```

Comment:
@pyoner we already have `serializer` function, why do we need a special package? 🤔